### PR TITLE
API-10264 Surveys renamed to Forms in v3.5

### PR DESCRIPTION
**Pull request by lucas-sajdak**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

- [Jira](https://livechatinc.atlassian.net/browse/API-11192)

# Description

In v3.5 we are renaming `surveys` to `forms`. Additionally the syntax of the response is changed so it is more clear (no more just magic IDs in repsonse)

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/data-reporting/reports-api/v3.5/index.mdx [hunk 1]
```diff
@@ -596,13 +596,22 @@ See [available parameters and filters](#available-parameters-and-filters). 💡
...
-| `records.{group_id}.{surveyid}.{answer_id}.{index}` | Number of times this index was selected for this Answer ID. |
+| Field                                                                            | Notes                                                    |
+| -------------------------------------------------------------------------------- | -------------------------------------------------------- |
+| `records`                                                                        | Contains `form` objects.                                 |
+| `records[{form_index}]`                                                          | Form at a given index.                                   |
+| `records[{form_index}].form_id`                                                  | Form ID.                                                 |
+| `records[{form_index}].count`                                                    | Number of times this form was filled.                    |
+| `records[{form_index}].group_id`                                                 | Group ID in which is this `form`.                        |
```
Comment (kamilatokarska) [inline src/pages/data-reporting/reports-api/v3.5/index.mdx:605]:
```suggestion
| `records[{form_index}].group_id`                                                 | Group ID in which this `form` is.                        |
```

File: src/pages/data-reporting/reports-api/v3.5/index.mdx [hunk 2]
```diff
@@ -596,13 +596,22 @@ See [available parameters and filters](#available-parameters-and-filters). 💡
...
+| `records[{form_index}].fields                                                    | Contains `field` objects.                                |
```
Comment (kamilatokarska) [inline src/pages/data-reporting/reports-api/v3.5/index.mdx:606]:
```suggestion
| `records[{form_index}].fields`                                                    | Contains `field` objects.                                |
```

File: src/pages/data-reporting/changelog/index.mdx [hunk 3]
```diff
@@ -32,6 +32,10 @@ The developer preview version provides a preview of the upcoming changes to the
 
 - There's a new method, [**Tags Chat Usage**](/data-reporting/reports-api/v3.5/#chat-usage), which shows how many chats are marked with each listed tag.
 
+### Chats
+
+- The `surveys` method has been renamed to [**Forms**](/data-reporting/reports-api/v3.5/#forms), which shows the number of submitted chat forms along with the count of specific answers.
```
Comment (kamilatokarska) [inline src/pages/data-reporting/changelog/index.mdx:37]: I suggest writing this info as two sentences to make it easier to read :)
`The `surveys` method has been renamed to [**Forms**](/data-reporting/reports-api/v3.5/#forms). It shows the number of submitted chat forms along with the count of specific answers.`

**Review by kamilatokarska — COMMENTED** [review]
A few corrections needed :)

File: src/pages/data-reporting/changelog/index.mdx (hunk 3 above)
Comment (lucas-sajdak) [reply src/pages/data-reporting/changelog/index.mdx:37]: Generally I agree... but I took an example from documentation about renaming method in v3.4 :) I will improve it

Comment (kamilatokarska) [reply src/pages/data-reporting/changelog/index.mdx]: Both are correct, but let's make it easier read ;)

**Review by kamilatokarska — APPROVED** [review]
:) 
Please let me know when it is ready to merge, and I will take care of it :)

**Pull request merged**